### ISSUE TITLE
Private (non-advertised) archives

### ISSUE THREAD

**Issue by julianpistorius** [beakerbrowser/hashbase#92]
**Problem**

Sometimes I want to share archives with collaborators when I'm offline, but don't want to advertise the presence of these archives.

At the moment when you view a user's profile on the Hashbase site you see all his activity, and all his archives:

![hashbase_pfrazee_archives](https://user-images.githubusercontent.com/742633/36766111-4bdc08b2-1c99-11e8-81ca-a1f38a0414f9.png)

I'd love to be able to mark certain archives as 'non-advertised' or 'non-discoverable'.

I would have to give the Hashbase URL to the party I want to grant access.

In other words for some archives I only want the reliability part of _Reliability and discovery!_™

**Concrete use cases**

- Collaborating on academic papers during the pre-publication phase
- Sharing photos or videos with family & friends
- Semi-private blogs/social media feeds

Two potential variations on this feature, increasing in levels of privacy:

1. Don't show the presence of the archive or any activity relating to it on the profile page, but still serve it on subdomains over `https://` and `dat://`
2. In addition, don't serve the archive on a subdomain - neither `dat://` or `https://` - but only via the the raw `dat://<hash>` URL (to prevent leaks via DNS)

**Comment by pfrazee** [comment]
Yeah @taravancil has been asking for this for a while. I'm +1, we'll add it to our roadmap

**Comment by erangell** [comment]
You will need to clarify somewhere your policy for technical support when a user needs to share an archive with a support person for the purpose of debugging.  That support person may need to look at files on hashbase, but should be bound by an agreement that they may not disclose any information about private user content, or retain any user content on any device after the ticket is closed.

**Comment by erangell** [comment]
See this page as an example:
https://www.startpage.com/eng/support-privacy-policy.html#hmb
